features/prompt-registry.mdx

```diff
@@ -32,30 +32,20 @@ It’s simple.
 <CodeGroup>
 
 ```python Python SDK
-template_dict = promptlayer.templates.get(prompt_name="my_template")
-```
-
-```python Python SDK (Legacy)
-template_dict = promptlayer.prompts.get("my_template")
+template_dict = promptlayer.templates.get("my_template")
 ```
 
 ```js JavaScript
 const template_dict = await promptlayer.templates.get("my_template");
 ```
 
-```js JavaScript (Legacy)
-
-```js JavaScript (Legacy)
-const template_dict = await promptlayer.prompts.get({ prompt_name: "my_template" });
-```
-
 </CodeGroup>
 
 Alternatively, use the REST API endpoint `/rest/get-prompt-template` ([read more](/reference/get-prompt-template)).
 
-_Note:_ We have a new prompt template schema that is more flexible and powerful. If you are using the new schema, you can use the `promptlayer.templates.get` method to get a template. If you are using the old schema, you can use the `promptlayer.prompts.get` method to get a template. Notice how `promptlayer.prompts.get` will always return templates in legacy shape, while `promptlayer.templates.get` will always return templates in the new shape.
+*Note:* We have deprecated the legacy prompt template schema. Use `promptlayer.templates.get` to retrieve prompt templates. Notice how `promptlayer.prompts.get` will always return templates in legacy shape. 
```

Comment:
Note: Please use `promptlayer.templates.get` to access prompt templates. The `promptlayer.prompts.get` function is deprecated and returns templates in the legacy format.